src/python/ensembl/io/genomio/genbank/extract_data.py

```diff
@@ -55,11 +55,12 @@ class UnsupportedData(Exception):
 
 
 class GenomeFiles(dict):
-    """Store the representation of the genome files created."""
+    """
+    Store the representation of the genome files created.
+    """
 
-    def __init__(self, out_dir: PathLike = Path.cwd()) -> None:
+    def __init__(self, out_dir: PathLike) -> None:
         super().__init__()
-        out_dir = Path(out_dir)
```

Comment:
This `Path()` line is required, to use the out_dir / "name" operations below, because PathLike itself doesn't support it.

Reply:
oh thats why mypy was failing, thanks